src/features/dataset/state/datasetState.ts

```diff
@@ -27,15 +34,23 @@ export const selectSessionUserCanEditDataset = (state: RootState): boolean => {
 
 export interface DatasetState {
   dataset: Dataset
+  datasetVersionInfo: VersionInfo
```

Comment:
this can just be `versionInfo`

Reply:
this should actually be called `header` & we should add a note:
```
// TODO(boandriy): header is currently a `VersionInfo`, but will have its own data structure in future iterations
```